### PR TITLE
Feature/oui skeleton

### PR DESCRIPTION
**Pull request by varun257**
add oui skeleton styles

JIRA: UU-845

### CODE REVIEW

File: packages/oui-skeleton/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```less
+  @import 'oui-skeleton/skeleton';
+```
+
+## Usage
+
+
+```html:preview
+<div class="oui-skeleton">
+    <span class="oui-skeleton__loader" style="width: 30%"></span>
```
Comment (AxelPeter) [inline packages/oui-skeleton/README.md]: No inline style in our preview, that could be copy/pasted.
We have classnames helpers for that. Check for `oui-doc-preview-only` and `oui-doc-preview-only-keep-children`.

File: packages/oui-skeleton/_variables.less [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@oui-skeleton-background-color: @oui-color-concrete;
+@oui-skeleton-background-image: linear-gradient(-90deg, rgba(250,250,250,0.80) 0%, @oui-color-nurse 46%, @oui-color-nurse 61%, rgba(242,242,242,0.80) 100%);
+@oui-skeleton-background-size: 50% 200px;
```
Comment (AxelPeter) [inline packages/oui-skeleton/_variables.less]: No px unit -> use rem-calc()

**Comment by varun257** [comment]
@AxelPeter  Have fixed the issues

File: packages/oui-skeleton/_mixins.less [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#oui {
+  .skeleton-sizes() {
```
Comment (neolitec) [inline packages/oui-skeleton/_mixins.less]: I think it's useless.

Comment (AxelPeter) [reply packages/oui-skeleton/_mixins.less]: We could have both. See my comment: https://github.com/ovh-ux/ovh-ui-angular/pull/185 :)

**Pull request merged**